cds_dojson/matcher.py

```diff
@@ -42,3 +46,45 @@ def query(self):
     def match(self, record, user_info=None):
         """Return True if record match the query."""
         return self.query.accept(MatchUnit(record))
+
+
+def matcher(record, entry_point):
```

Comment:
maybe you can rename `entry_point` with `entry_point_group`.
It should be more clear (and there is no ambiguity with the `entry_point` inside the `foreach`).